Makefile

```diff
@@ -21,3 +21,12 @@ container: clean build
 
 container.push: container
 	docker push ${HUB}/header-append-filter:${VERSION}
+
+.PHONY: lint
+# TODO: Add linter for rust code
+lint:
+	find . -name '*.sh' -print0 | xargs -0 -r shellcheck
```

Comment:
This doesn't look right. Linter for rust code looking for sh files and running shellcheck?

Reply:
This isn't linter for rust. It's a general linter. Currently only shell is implemented. Look at the todo item above.